include/RAJA/pattern/kernel/Lambda.hpp

```diff
@@ -86,6 +87,17 @@ template<typename Head, typename...Tail>
 struct extractor
 {};
 
+template<camp::idx_t id, typename...Tail>
+struct extractor<RAJA::statement::OffSet<id>, Tail...>
+{
+  template<typename Data>
+  static auto extract_arg(Data &&data) ->
+    decltype(camp::get<id>(data.offset_tuple))
+  {
```

Comment:
@ajkunen , I added support for offsets. Just wanted to double check if this is what you had in mind. If so I may just need some unit test now.